extensions/replication/failedCRR/FailedCRRConsumer.js

```diff
@@ -24,6 +24,7 @@ class FailedCRRConsumer {
         this.logger = new Logger('Backbeat:FailedCRRConsumer');
         this._failedCRRProducer = new FailedCRRProducer(this.kafkaConfig);
         this._backbeatTask = new BackbeatTask();
+        this._statsClient = new StatsModel(redisClient);
```

Comment:
Is this ok to set without passing interval and expiry as params?

Reply:
Should be okay. It's not asserted in the constructor, and the only method used from that class  here is `addToSortedSet` which does not need the interval or expiry values.